### PR TITLE
File locations changes, PHP SDK updated

### PR DESCRIPTION
**Pull request by tarampampam**
| Q             | A
| ------------- | ---
| Bug fix?      | Yes
| New feature?  | Yes

### Added

- Json-schemas for each json file in `./data` directory

### Changed

- File locations:
  - `./data/auto_categories/auto_categories.json` &rarr; `./data/vehicle/categories.json`
  - `./data/vehicle_types/vehicle_types.json` &rarr; `./data/vehicle/types.json`
  - `./data/auto_fines/auto_fines.json` &rarr; `./data/vehicle/fine/articles.json`
  - `./data/auto_regions/auto_regions.json` &rarr; `./data/subject/codes.json`
  - `./data/cadastral_districts/cadastral_districts.json` &rarr; `./data/cadastral/districts.json`
  - `./data/registration_actions/registration_actions.json` &rarr; `./data/vehicle/registration/actions.json`
  - `./data/repair_methods/repair_methods.json` &rarr; `./data/vehicle/repair/methods.json`
- Object properties `districts` renamed to `areas` in `./data/cadastral/districts.json`
- Object properties `code` and `areas.*.code` now integers in `./data/cadastral/districts.json`
- (PHP SDK) Methods in `StaticReferencesData` renamed and does not accept any arguments:
  - `::getAutoCategories()` &rarr; `::vehicleCategories()`
  - `::getAutoRegions()` &rarr; `::subjectCodes()`
  - `::getRegistrationActions()` &rarr; `::vehicleRegistrationActions()`
  - `::getRepairMethods()` &rarr; `::vehicleRepairMethods()`
  - `::getAutoFines()` &rarr; `::vehicleFineArticles()`
  - `::getVehicleTypes()` &rarr; `::vehicleTypes()`
  - `::getCadastralDistricts()` &rarr; `::cadastralDistricts()`
- (PHP SDK) Return value type for methods `->getHash()` and `->getFilePath()` in `StaticReferenceInterface`

### Removed

- Object properties `short`, `okato` and `type` removed from `./data/subject/codes.json`
- (PHP SDK) `StaticReference::getContent()` and `StaticReferenceInterface::getContent()`

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I wrote unit tests for my code _(if tests is required for my changes)_
- [x] I have made changes in `CHANGELOG.md` file

### CODE REVIEW

File: sdk/php/src/ReferencesData/StaticReferenceInterface.php [hunk 1]
```diff
@@ -11,35 +11,24 @@ interface StaticReferenceInterface
...
      * Returns source file path.
      *
      * @return string
      */
-    public function getFilePath();
+    public function getFilePath(): string;
 
     /**
-     * Returns source content as an array.
+     * Returns source content as abjects or associative arrays.
```
Comment (Reallife) [inline sdk/php/src/ReferencesData/StaticReferenceInterface.php]: Typo `abjects` => `objects`

File: sdk/php/tests/ReferencesData/StaticReferenceTest.php [hunk 2]
```diff
@@ -71,7 +71,7 @@ public function testGetFilePath(): void
      */
     public function testGetContent(): void
```
Comment (Reallife) [inline sdk/php/tests/ReferencesData/StaticReferenceTest.php]: Not important but `public function testGetData(): void` may be?

Comment (tarampampam) [reply sdk/php/tests/ReferencesData/StaticReferenceTest.php]: done

File: sdk/php/src/ReferencesData/StaticReferenceInterface.php (hunk 1 above)
Comment (tarampampam) [reply sdk/php/src/ReferencesData/StaticReferenceInterface.php]: done

File: data/vehicle/fine/articles.schema.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "array",
+    "minLength": 1,
+    "items": {
+        "type": "object",
+        "properties": {
+            "article": {
+                "type": "string",
+                "minLength": 1,
+                "pattern": "\\d{1,2}\\.\\d{1,2}\\s?[\\d\\w]?"
```
[the remark this answers is not in the document]
Comment (tarampampam) [reply data/vehicle/fine/articles.schema.json]: done! (`\d{1,2}\.\d{1,2}[\s\wа-яёА-ЯЁ'.]*`)

**Pull request merged**